src/scheduler/pbs_sched.c

```diff
@@ -1445,6 +1450,23 @@ main(int argc, char *argv[])
 		}
 	}
 
+	/* Initialize cleanup lock */
+	if (pthread_mutexattr_init(&attr) != 0) {
```

Comment:
It seems like this block of code is also used in multi_threading.c. Can this be made into a common function to generate recursive mutex?